CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing to the Project
+
+Hey there! Welcome to the project! Here's some things to keep in mind before you get started.
+
+# Questions
+
+If you have a question, and there isn't any specific Issue that the question is about, please post your questions in the [discussion](https://github.com/orgs/alveusgg/discussions).
+
+# Issues
+
+### Bug Reports:
+
+When reporting a bug, please open an issue following this format:
+
+- `Title`
+- `Expected Behavior`
+- `Current Behavior`
+- `Any Relevant Screenshots`
+- Go to [what's my browser](https://www.whatsmybrowser.org/), copy the link generated and paste it into the issue description. E.g. `https://www.whatsmybrowser.org/b/WTBQ3`
+- Add the `bug` label
+
+### Ideas / Enhancements
+
+When sharing an idea or enhancement open an issue in the following format:
+
+- `Title`
+- `Description` of the idea
+- `Justification` of why this idea would be beneficial
+- Add the `idea` or `enhancement` label.
+
+# Development
+
+### Dev Environment
+
+- To start a local dev environment, follow this [guide](https://github.com/alveusgg/alveusgg#how-to-develop--getting-started).
+- To setup a production instance of the website, follow this [guide](https://github.com/alveusgg/alveusgg#how-to-set-up-your-own-production-instance)
+- Overview of architecture, refer to [systems overview](https://github.com/alveusgg/alveusgg#systems-overview).
+- More detailed overview the architecture, refer to [#9](https://github.com/alveusgg/alveusgg/issues/9).
+
+### Commit Messages
+
+- Try to keep your commits to small units of change. Prefering more commits over less.
+- Please keep your commit message to ae a concise message that clearly explains the change made.
+
+### PRs
+
+- `Title`
+- `Description`
+- `Notes for testing the change`
+- `Reference to the issue the PR fixes if applicible`
+- Please add one of the following Github [keywords](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue) (e.g. `Closes #13`) in the related issue so that it automatically closes.
```

Comment:
Similar to the issues note, we can create a [PR template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository) (just Markdown, not a form) instead of having this in text.

Reply:
Cool, I'll look into these and see if I can get it setup.